spec/js/extend.spec.js

```diff
@@ -0,0 +1,41 @@
+var I18n = require("../../app/assets/javascripts/i18n")
+  , Translations = require("./translations")
+;
+
+describe("Extend", function () {
```

Comment:
Can we add a test case for "NOT changing the original input objects"?
Well since jQuery one does that :S